app/helpers/application_helper.rb

```diff
@@ -1,2 +1,13 @@
 module ApplicationHelper
+  MAX_VOTES = 3
+
+  private
+  def user_votes
+    (cookies["#{@hackday.id}_votes"] || MAX_VOTES).to_i
```

Comment:
the reason that its bad form is that its going to pollute the name space everywhere and its not obvious where to find this function.